web/src/left_sidebar_navigation_area.js

```diff
@@ -154,6 +155,7 @@ function do_new_messages_animation($li) {
 }
 
 export function highlight_inbox_view() {
+    pm_list.deselect_dm_list_item();
```

Comment:
Instead of using the ```deselect_dm_list_item()``` function , I think you should have a look at ```close()``` function in ```pm_list.js``` as it is what the issue needs to resolve.
```suggestion
    pm_list.close();
```

Reply:
@shashank-23002 Thank you so much for your review, As i am looking in [CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/lagging.2C.20lingering.20DM-row.20highlight) most of the conversations revolves around the highlight linger, I might have misunderstood something somewhere. Thank you so much for the Review once again I'll work on my patch and revert back ASAP.

Reply:
@shashank-23002 Thank you so much for the Tip, I have looked into the close function and I do see that it is what i need, my question is, do i call it only when narrowed to a stream or when any other row in the leftside bar is clicked